README.md

```diff
@@ -1,11 +1,36 @@
 # OpencC2Test
 
+OpenC2 is a standardized language for controlling cyber defense technologies. It allows different systems and components to communicate and coordinate with each other, regardless of the specific products or technologies they use. This enables automated and efficient management of cyber defenses.
+
+## Setup Guide
+
 To start your Phoenix server:
 
   * Run `mix setup` to install and setup dependencies
+  * Source the environment variables by running `source .env`. This contains environment specific settings and keys.
   * Start Phoenix endpoint with `mix phx.server` or inside IEx with `iex -S mix phx.server`
 
-Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
+## How to use the project
+
+Now you can visit [`localhost:4000`](http://localhost:4000) from your browser. Follow the following steps to run your project.
+
```

Comment:
Let it be known that we are currently testing only one device, TwinklyMaha, and provide a link to redirect users to the project to start testing.